### ISSUE TITLE
Check time-varying $r$

### ISSUE THREAD

**Issue by jlperla** [econtoolkit/continuous_time_methods#36]
Want the $r$ to change smoothly.

Check for some interior set of the $x$ values (e.g  i in the middle half of the statespace).  FInd the
$$
\max_{1/4 I < i < 3/4 I}\{v^{n+1}_i - v^n\}
$$
and see how that number changes with $n$

**Comment by sevhou** [comment]
I tried for time varying $u$ function, the value $\max_{n,i} v_{i}^{1}-v_{i}^{n}=7.4$, with tgrid 0 to 1, zgrid from 0.1 to 3, it's a strictly increasing function on t dimension.

**Comment by jlperla** [comment]
Are there any odd discontinuities on the final few points?  Play around
with this, maybe even doing a 3D plot of the value function over time.

You should be able to do that we a `reshape` command in matlab, and then
`surf` or something like that.

**Comment by sevhou** [comment]
Yeah I've done that and also plot some 2-D graphs and check along t dimension and z dimension, I think it behaves smoothly.

**Comment by jlperla** [comment]
OK.  Keep all of the code in the tests for later comparison.  If things
seem to be smooth for both the $r$ and the $u$ changes, that is good news.
Make sure you are looking at examples where the changes are cranked up to
be large enough to see relatively large swings.
